src/ch/ehi/ili2db/base/Ili2db.java

```diff
@@ -1197,6 +1198,12 @@ public static void runSchemaImport(Config config,String appHome)
                         if(conn!=null) {
                             MetaAttrUtility.addMetaAttrsFromDb(td, conn, config.getDbschema());
                         }
+                        if(importToDb){
+                            TypeUtility.writeColumnProps(conn, config.getDbschema(), mapping);
+                            if(config.getCreateTypeConstraint()){
+                                TypeUtility.addTypeConstraints(conn, config.getDbschema(), mapping);
```

Comment:
refactor, so that constraints are written where the t_type column is created (FromIliRecordConverter.generateTable()). And I will add a function DbColVarchar.setValueRestriction(String[] possibleValues) in ehisqlgen.jar